### DISCUSSION TITLE
Red square around Russian letters

### DISCUSSION THREAD

**Discussion by Gentostage** [coder/code-server#4720] [Q&A] [answered]
I am deploy code-server on linux machine, but when I type Russian letters, it highlights certain ones with a red square.
Is there any way to fix this?
![image](https://user-images.githubusercontent.com/34418402/148840941-5fda35b2-7905-46b2-af23-69af5791ff76.png)

**Comment by lg-nineoneninethreenine** [comment, marked as answer]
Hey,
Can you check your Settings(_<kbd>F1</kbd>_ > _Preferences: Open Settings (UI)_ > _Keyboard_)? It might be speaking language syntax highlighting. If not, can you type `code-server --version` in your terminal and send us the console log?

**Reply by Gentostage** [reply]
Thank you, once again I decided to search in the settings. And found the parameter.
![image](https://user-images.githubusercontent.com/34418402/148890303-72fed716-454b-489d-97da-16415c94b2bb.png)

**Reply by jsjoeio** [reply]
Hooray! Thanks for helping out @im-coder-lg
